WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,3 +1,6 @@
-- “Get to know your app” flows now stay on screen after rotating your device.
-- Long titles display properly on Stats cards without being cut off.
-- We added the Enter key to Post excerpts to allow for easy line breaks.
+* [***] [internal] Updates the target sdk to 31 - Android 12 [https://github.com/wordpress-mobile/WordPress-Android/pull/17153]
+* [*] Updated About screen to use correct urls. [https://github.com/wordpress-mobile/WordPress-Android/pull/17282]
```

Comment:
@ravishanker I took the liberty to reduce the priority indicator of this one from the `[***]` you used in your PR to just `[*]`, as I feel such change won't impact end users' use of the apps that much, and besides, the other Release Notes item about the Update of splash screen (just below), which seems like a very similar "small design update" change, was also `[*]`. See also https://github.com/wordpress-mobile/WordPress-Android/pull/17282#discussion_r996839120

Since during code freeze and release notes editorialization we split the release notes between WP and JP to be editorialized separately, I also tried to come up with a dedicated wording that didn't mention the "for WordPress and Jetpack apps to their respective urls" wording so that each separate final release notes would make sense on its own. Open to other suggestions for that wording (which will be re-worded and editorialized by our freelance writer anyway, but it might be good to provide her with a good initial draft to help her nail it 🙂 )